### ISSUE TITLE
Vanilla Nerf configuration

### ISSUE THREAD

**Issue by wangyida** [dunbar12138/DSNeRF#68]
Thanks for your interesting work, I tried to train the model with fern_2v.txt which works reasonably. But the model can not be trained when I set depth_loss = False and using the Nerf loss to regress RGB alone. There are only white images produced in this case. Can you inform me whether there is a proper configuration which works well for training without depth_loss, so that I can compare against the baseline methods.

Thanks

**Comment by dunbar12138** [comment]
Hi, thanks for your interest!

Disabling the depth_loss in the config file should be a reasonable baseline for vanilla NeRF and shouldn't provide only white images.

Is fern_2v.txt the config you were using?
